deploy/example/deployment.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   resources:
     requests:
       storage: 1Gi
-  storageClassName: blobfuse.csi.azure.com
+  storageClassName: blob
```

Comment:
blobfuse-csi?

Reply:
there is no blob in-tree driver, also we are going to develop blob based NFS service, use `blob` should make more sense